### PR TITLE
Create CNC SIG Charter

### PR DESCRIPTION
**Pull request by 0rdinary-J0e**
Updated Co-Leads, Authorization, and removed elections.

### CODE REVIEW

**Comment by 0rdinary-J0e** [comment]
Fixed Typo

**Review by chungl — APPROVED** [review]
This charter was conditionally approved at the [October board meeting](https://denhac.org/board-minutes/board-meeting-2024-10-01) with the following stipulations:

```
SIG CNC charter @Joseph S
Read the proposed charter here.

There are issues:

There should just be 2 co-chairs, that way there is a standardization across our SIGs. 

The process for an election for co-chairs is too cumbersome. It's fine to have elections to appoint these people, but we shouldn't be rigid about it in the charger. 

The SIG should have the power to revoke authorization. 

Board votes 5-0 to provisionally approve the charter, pending the approval of the pull request by 2 board members. 
```

The issues have been addressed. This charter will be eligible to be merged with a second board approval.

File: sigs/sig-cnc-charter.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+The CNC SIG will be led by two Co-Leads, and a group of appointed Trainers/Maintainers. 
+
+General responsibilities for these Roles (this is not an exhaustive list):
+
+* The SIG Co-Leads will be generally responsible for SIG project management and/or delegation of these responsibilities, as well as the general governance of the group's actions at large.
+* SIG Co-Leads are responsible for the appointments of properly qualified Maintainers and Trainers, as well as providing any extended training required for both.
+* Maintainers will support the Co-Leads in matters regarding the upkeep and management of machines.
+* Trainers will be responsible for authorizing members to use the machines safely and providing assistance with machine usage when appropriate.
+* There is not a fixed number of Maintainer and Trainer positions available; they are filled based on demonstrated skill, merit, enthusiasm, and member demand for the machines.
+* When changing leadership, responsibilities should be officially transitioned to the new party on the first day of the month following an election for new leadership, or as privately discussed between the previous and incoming leadership.
```
Comment (anne-decusatis) [inline sigs/sig-cnc-charter.md]: please remove reference to elections - could reword as: "When changing leadership, responsibilities should be officially transitioned to the new party on the first day of the month following the change in leadership, or as privately discussed between the previous and incoming leadership."
but I think removing this line entirely would be fine too

File: sigs/sig-cnc-charter.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+#### ***Out-of-Scope***
+
+* Maintenance and upkeep of member-owned CNC machines that aren't for use by denhac members
+* Member-owned materials
+* Anything not specifically listed as In-Scope, or as determined by the SIG Leads
+
+## Organization Management
+
+SIG decisions will officially be made by majority vote between the SIG leads who should strongly take into consideration the sentiments of any denhac member with a special interest in the CNC routers on site, while ultimately acting in the best interests of denhac as a whole. SIG leads are encouraged to make decisions using majority votes from CNC SIG members. Decisions made by the CNC SIG leads can always be overridden by the denhac board of directors.
```
Comment (anne-decusatis) [inline sigs/sig-cnc-charter.md]: could use rewording at "SIG decisions will officially be made by majority vote between the SIG leads" - there are 2 leads, what's a majority vote between 2 people?

suggest reword to: SIG decisions will be made by the SIG leads, who should strongly take into consideration...

**Review by anne-decusatis — COMMENTED** [review]
two small wording comments related to the cleanups in word choice around elections and number of leads, I will approve when addressed.

**Review by anne-decusatis — APPROVED** [review]
thanks for addressing my comments! LGTM

**Pull request merged**